recipes/kaolin/build.sh

```diff
@@ -0,0 +1,22 @@
+if [[ "$cuda_compiler_version" == "None" ]]; then
+  export FORCE_CUDA=0
+else
+  export TORCH_CUDA_ARCH_LIST="3.5;5.0+PTX"
+  if [[ ${cuda_compiler_version} == 9.0* ]]; then
+      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;7.0"
+  elif [[ ${cuda_compiler_version} == 9.2* ]]; then
+      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0"
+  elif [[ ${cuda_compiler_version} == 10.* ]]; then
+      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5"
+  elif [[ ${cuda_compiler_version} == 11.0* ]]; then
+      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0"
+  elif [[ ${cuda_compiler_version} == 11.1* ]]; then
+      export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
```

Comment:
We may want to extend this to handle CUDA 11.2-11.4, but that can be done in the feedstock :)